src/core/lv_refr.c

```diff
@@ -474,6 +480,117 @@ lv_obj_t * lv_refr_get_top_obj(const lv_area_t * area_p, lv_obj_t * obj)
 }
 
 
+void lv_obj_refr(lv_layer_t * layer, lv_obj_t * obj)
```

Comment:
This function became public so it deserves parameter validation, using LV_ASSERT would be sufficient.